guides/intro-to-ploomber/README.md

```diff
@@ -0,0 +1,151 @@
+<!-- start header -->
+To run this example locally, [install Ploomber](https://docs.ploomber.io/en/latest/get-started/quick-start.html) and execute: `ploomber examples -n guides/first-pipeline`
+
+To start a free, hosted JupyterLab: [![binder-logo](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/ploomber/binder-env/main?urlpath=git-pull%3Frepo%3Dhttps%253A%252F%252Fgithub.com%252Fploomber%252Fprojects%26urlpath%3Dlab%252Ftree%252Fprojects%252Fguides/first-pipeline%252FREADME.ipynb%26branch%3Dmaster)
+
+Found an issue? [Let us know.](https://github.com/ploomber/projects/issues/new?title=guides/first-pipeline%20issue)
+
+Have questions? [Ask us anything on Slack.](https://ploomber.io/community/)
+
+For a notebook version (with outputs) of this file, [click here](https://github.com/ploomber/projects/blob/master/guides/first-pipeline/README.ipynb)
+<!-- end header -->
+
+
+
+
+# Your first Python pipeline
```

Comment:
is this the file I should review? this looks very similar to the existing "first-pipeline", and I remember you said this new version would have the parallel execution and report generation stuff